app/models/topic.rb

```diff
@@ -1,9 +1,22 @@
 class Topic < ApplicationRecord
+  enum status: { opening: 0, locked: 5, deleted: 10 }
```

Comment:
sao lại để trạng thái như thế này nhỉ? Giải thích cho a ở đây hoặc mai cũng đc

Reply:
tính mở rộng, đề phòng trường hợp cần thêm enum vào giữa `opening` hoặc `locked`. Nếu sử dụng default là 0, 1, 2 thì sẽ phải sửa toàn bộ db nếu muốn nâng cấp.